test/e2e/auth/node_authz.go

```diff
@@ -179,4 +181,18 @@ var _ = SIGDescribe("[Feature:NodeAuthorizer]", func() {
 		err := c.CoreV1().Nodes().Delete("foo", &metav1.DeleteOptions{})
 		Expect(apierrors.IsForbidden(err)).Should(Equal(true))
 	})
+
+	It("The kubelet's main port 10250 should fail with the Forbidden error", func() {
+		ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
+		defer cancel()
+
+		_, err := c.CoreV1().RESTClient().Get().
+			Context(ctx).
+			Resource("nodes").
+			SubResource("proxy").
```

Comment:
same comment, this is likely testing the apiserver, not the kubelet. I'm also surprised the client was rejected here... the framework client is typically highly privileged